package.json

```diff
@@ -10,6 +10,7 @@
     "serve": "ws --spa index.html -d ./dist",
     "stylelint": "stylelint './src/**/*.scss' ",
     "stylelint:fix": "stylelint './src/**/*.scss' --fix",
+    "test": "ts-node -r ignore-styles tests/**/*.ts",
```

Comment:
One more thing that might help is installing `tsconfig-paths` for absolute imports in our components. Then add `-r tsconfig-paths/register`